cpp/src/Ice/Network.cpp

```diff
@@ -113,7 +113,8 @@ namespace
         return fd;
     }
 
-    vector<Address> getLocalAddresses(ProtocolSupport protocol, bool includeLoopback, bool singleAddressPerInterface)
```

Comment:
We only call this function once with includeLoopback=true and singleAddressPerInterface=true

Reply:
Very nice simplification.
Crazy we still had these 2 bools sitting around...